### PR TITLE
feat: duplicate capture v1: display distance between two selected captures

### PR DESCRIPTION
**Pull request by amesk3**
## Description
In an effort to distinguish duplicate captures, this feature gets the distance between two currently selected features.

**Issue(s) addressed**
Duplicate verification issue.

- Resolves #issue_number
[Issue 781 ](https://github.com/Greenstand/treetracker-admin-client/issues/781)

**What kind of change(s) does this PR introduce?**

- [x] Enhancement
- [ ] Bug fix
- [ ] Refactor

**Please check if the PR fulfils these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**
N/A

**What is the new behavior?**
On /verify, when two selections are selected, Side Panel displays the distance between the two selections based off of their lat/lon. There is now label beneath the Select All and Select None buttons: Distance between selected captures: x.x m when two captures are selected

## Breaking change

**Does this PR introduce a breaking change?**

## Other useful information
This is the first part of addressing the duplicate image verification issue.

### CODE REVIEW

**Comment by amesk3** [comment]
image
image
image
Different scenarios of there being nothing selected, 2 captures selected, and more than 2 captures selected.

File: src/components/SidePanel.js [hunk 1]
```diff
@@ -118,6 +119,26 @@ function SidePanel(props) {
...
+      {
+        latitude: Number(obj1.lat),
+        longitude: Number(obj1.lon),
+      },
+      {
+        latitude: Number(obj2.lat),
+        longitude: Number(obj2.lon),
+      }
+    );
+    return `Distance bewteen selected captures: ${distance.toLocaleString()}m`;
```
Comment (nmcharlton) [inline src/components/SidePanel.js:139]: Let's limit this distance to one decimal place.

**Review by nmcharlton — CHANGES_REQUESTED** [review]
There are a couple of tests that are now failing. Could you take a look? They may need updating to reflect the changes you've made.
```
npm run test
```

File: src/components/SidePanel.js (hunk 1 above)
Comment (amesk3) [reply src/components/SidePanel.js]: Done!

**Review by nmcharlton — APPROVED** [review]
Nice work!

**Pull request merged**